In [23]:
import pandas as pd
import scipy.stats as stats

In [6]:
stats_test = pd.read_csv('Testing_Of_Hypothesis.csv')

In [7]:
stats_test

Sr No S_Cust_id  Samsung Score Apple_Cust_id  Apple_Score
0       1   S336499             94       A336525           82
1       2    S73004             99        A73011           83
2       3    S79537             74        A79538           88
3       4   S769168             74       A769222           80
4       5   S609791             83       A609801           74
..    ...       ...            ...           ...          ...
95     96   S285617             77       A285629           87
96     97   S324135             85       A324151           82
97     98   S292076             83       A292091           88
98     99   S936528             82       A936566           91
99    100   S489010            100       A489031           95

[100 rows x 5 columns]

### Q1.The Samsung Average customer satisfaction score is minimum 75%.

In [15]:
stats_test.rename(columns = {'Samsung Score':"Samsung_Score"},inplace = True)

In [21]:
stats_test.Samsung_Score

0      94
1      99
2      74
3      74
4      83
     ... 
95     77
96     85
97     83
98     82
99    100
Name: Samsung_Score, Length: 100, dtype: int64

In [27]:
stats.ttest_1samp(a = stats_test.Samsung_Score, popmean = 75)

Ttest_1sampResult(statistic=-8.527141742860234, pvalue=1.758901525193106e-13)

- Significance Level @ 0.05

- t value = 8.358287450922418

- p value = 4.0739777034636007e-13

- p value is smaller than significance level

- Accept Alternative

- Population mean != Sample Mean

- **"The Samsung Average customer satisfaction score is not minimum 75%."**


### 2. The overall average satisfaction score of Samsung is same as Apple. There is no significant difference in the satisfaction scores. 

In [28]:
stats.ttest_rel(a = stats_test.Samsung_Score, b = stats_test.Apple_Score)

Ttest_relResult(statistic=-1.9015800946572132, pvalue=0.06013359092250471)

- Significance Level @ 0.05

- t value = -1.90

- p value = 0.060

- p value is greater than significance level

- Accept Null

- Sample 1 mean = Sample 2 Mean

- **"The overall average satisfaction score of Samsung is same as Apple. There is no significant difference in the satisfaction scores."**